v2.1/configure-zone.md

```diff
@@ -0,0 +1,111 @@
+---
+title: CONFIGURE ZONE
+summary: Use the CONFIGURE ZONE statement to add, modify, reset, and remove replication zones.
+toc: true
+---
+
+<span class="version-tag">New in v2.1:</span> Use `CONFIGURE ZONE` to add, modify, reset, and remove [replication zones](configure-replication-zones.html).
+
+In CockroachDB, you can use **replication zones** to control the number and location of replicas for specific sets of data, both when replicas are first added and when they are rebalanced to maintain cluster equilibrium.
+
+## Synopsis
+
+**alter_zone_range_stmt ::=**
+
+<div>
+  {% include {{ page.version.version }}/sql/diagrams/alter_zone_range.html %}
+</div>
+
+**alter_zone_database_stmt ::=**
+
+<div>
+  {% include {{ page.version.version }}/sql/diagrams/alter_zone_database.html %}
+</div>
+
+**alter_zone_table_stmt ::=**
+
+<div>
+  {% include {{ page.version.version }}/sql/diagrams/alter_zone_table.html %}
+</div>
+
+**alter_zone_index_stmt ::=**
+
+<div>
+  {% include {{ page.version.version }}/sql/diagrams/alter_zone_index.html %}
+</div>
+
+## Required privileges
+
+Currently, only the `root` user can configure replication zones.
+
+## Parameters
+
```

Comment:
Would add a note somewhere in here that replication zones for rows and indexes is enterprise-only.  I have some content to this effect in this PR: https://github.com/cockroachdb/docs/pull/3850  (Update: I see you do note it below, so perhaps not needed?  Still might be helpful while scanning the upper portion of the doc IMO.)